src/Parser/DisciplinePriest/Modules/Spells/Atonement.js

```diff
@@ -101,27 +102,17 @@ class Atonement extends Module {
       lastAtonementAppliedTimestamp: event.timestamp,
     };
     this.currentAtonementTargets = this.currentAtonementTargets.filter(id => id.target !== atonement.target);
-    debug && console.log(`%c${this.owner.combatants.players[atonement.target].name} lost an atonement`, 'color:yellow', this.currentAtonementTargets);
+    debug && console.log(`%c${this.owner.combatants.players[atonement.target].name} lost an atonement`, 'color:red', this.currentAtonementTargets);
     this.owner.triggerEvent('atonement_faded', event);
   }
 
   on_byPlayer_heal(event) {
     const spellId = event.ability.guid;
-    if (spellId !== SPELLS.ATONEMENT_BUFF.id) {
+    if ([SPELLS.ATONEMENT_HEAL_NON_CRIT.id, SPELLS.ATONEMENT_HEAL_CRIT.id].indexOf(spellId) === -1) {
       return;
     }
-    // if (!this.owner.toPlayer(event)) {
-    //   return;
-    // }
-    if (this.lastAtonementAppliedTimestamp === null) {
-      // This can be `null` when Atonement wasn't applied in the combatlog. This often happens as Discs like to apply Atonement prior to combat.
-      this.lastAtonementAppliedTimestamp = this.owner.fight.start_time;
-      debug && console.warn('Atonement: was applied prior to combat');
-    }
 
-    if ((event.timestamp - this.lastAtonementAppliedTimestamp) < (this.atonementDuration * 1000)) {
-      return;
-    }
+    event = Object.assign(event, { isAtonementHeal: true });
```

Comment:
I don't like this :( Whether a heal is an Atonement heal can be determined with a pure function; you could add a helper function named `isAtonementHeal(event)` that returns `true` if it is and otherwise `false`. This can be imported around to be reduce the duplication of `[SPELLS.ATONEMENT_HEAL_NON_CRIT.id, SPELLS.ATONEMENT_HEAL_CRIT.id].indexOf(spellId) === -1`. I think that would be a cleaner solution.

Adding properties to `event` is intended for information that requires state to be determined. An example of this would be the bolt count of Penance; you can only determine this by tracking something like a buff or the interval between casts, this requires some sort of state and can not be determined through a pure function.

Please revert this or change this to a different solution.